File: tests/conformance/fixtures/go/22-review-node.go
package main

import sykli "github.com/yairfalse/sykli/sdk/go"

func main() {
	p := sykli.New()

	p.Task("test").Run("go test ./...")
	p.Review("review-code").
		Primitive("lint").
		Agent("claude").
		Context("src/**/*.go").
		After("test").
		Deterministic(true)

	p.Emit()
}

File: tests/conformance/fixtures/typescript/22-review-node.ts
import { Pipeline } from '../../../../sdk/typescript/src/index';

const p = new Pipeline();

p.task('test').run('go test ./...');
p.review('review-code')
  .primitive('lint')
  .agent('claude')
  .context('src/**/*.go')
  .after('test')
  .deterministic(true);

p.emit();
